### DISCUSSION TITLE
Symfony aliases on WHM/Cpanel

### DISCUSSION THREAD

**Discussion by MarcAndreBelisle** [symfony/symfony#46236] [Q&A]
Hello, thank you in advance for any help, I've been trying for a few days now and I'm lost.

We currently have a WHM + cPanel host. With a main domain like **school.com**
Each students have a subdomain (with their own cPanel): **student1.school.com**

Now, each students are building a Symfony app in their subdomain: **student1.school.com/symfony**
To view their Website, we need to go to **student1.school.com/symfony/public**

In the old days, we used Wampserver with aliases but now with the WHM Include Editor, I can't seem to find what I need to do the same. I tried every .htaccess variation I found online but to no success.

In Wampserver we had for exemple:
```
Alias /student1/symfony "e:/inetpub/student1/symfony/public"
<Directory "e:/inetpub/student1/symfony/public">
    AllowOverride All
    Require all granted
</Directory>
```

What would be the WHM alternative for our current setup?

Thank you very much!

**Comment by stof** [comment]
this is a question to ask to the WHM support rather than to the Symfony support (I don't even know what WHM is btw)

**Reply by MarcAndreBelisle** [reply]
Hello Stof, WHM is simply the server manager on top of the cPanels. Where you manage accounts and system wide settings.
I could ask on the cPanel forum, but I believe it is more of a Symfony thing, as to what settings it needs to work.

Thank you.

**Reply by stof** [reply]
well, knowing how to customize the webserver configuration in WHM is definitely not a Symfony thing.

Symfony projects expects you to configure the webserver with the `public` folder as the document root, and each request that does not correspond to an existing file in the `public` folder being sent to the `index.php` file (which is basically the same requirement than other modern PHP frameworks). The way to do it for each server is then a support question regarding those servers.

**Reply by MarcAndreBelisle** [reply]
I worked a bit on this and I end up on a "No input file specified." page when I change this document root thing. Does that ring a bell?
